views/job-roles/new.njk

```diff
@@ -0,0 +1,180 @@
+{% extends "layout.njk" %}
+
+{% block title %}{{ title }}{% endblock %}
+
+{% block content %}
+<div class="container mx-auto px-4 py-8 max-w-4xl">
+  <div class="flex justify-between items-center mb-8">
+    <h1 class="text-5xl font-bold">{{ title }}</h1>
+    <a href="/jobs" class="btn btn-outline btn-md text-sm">
+      <svg xmlns="http://www.w3.org/2000/svg" class="h-4 w-4" viewBox="0 0 20 20" fill="currentColor">
+        <path fill-rule="evenodd" d="M9.707 16.707a1 1 0 01-1.414 0l-6-6a1 1 0 010-1.414l6-6a1 1 0 011.414 1.414L5.414 9H17a1 1 0 110 2H5.414l4.293 4.293a1 1 0 010 1.414z" clip-rule="evenodd" />
+      </svg>
+      Back to Job Roles
+    </a>
+  </div>
+
+  <div class="card bg-base-100 shadow-xl">
+    <div class="card-body">
+      <h2 class="card-title text-2xl mb-6">Job Role Information</h2>
+      
+      <form action="/jobs/new" method="POST" class="space-y-6">
+        <div class="grid grid-cols-1 md:grid-cols-2 gap-6">
+          <!-- Job Name -->
+          <div class="form-control">
+            <label class="label" for="name">
+              <span class="label-text text-lg font-semibold">Job Name *</span>
+            </label>
+            <input 
+              type="text" 
+              id="name"
+              name="name" 
+              placeholder="e.g., Software Engineer" 
+              class="input input-bordered input-lg w-full" 
+              required
+            />
+          </div>
+
+          <!-- Location -->
+          <div class="form-control">
+            <label class="label" for="location">
+              <span class="label-text text-lg font-semibold">Location *</span>
+            </label>
+            <select id="location" name="location" class="select select-bordered select-lg w-full" required>
+              <option disabled selected>Select location</option>
+              <option value="Belfast">Belfast</option>
+              <option value="London">London</option>
+              <option value="Manchester">Manchester</option>
+              <option value="Birmingham">Birmingham</option>
+              <option value="Edinburgh">Edinburgh</option>
+              <option value="Leeds">Leeds</option>
+              <option value="Glasgow">Glasgow</option>
+              <option value="Remote">Remote</option>
```

Comment:
instead of hardcoding it in the view itself, it would be better to pass a list of options from the controller, and here just iterate through with nunjucks to create the appropriate html